python/cudf/cudf/_lib/pylibcudf/copying.pxd

```diff
@@ -13,3 +15,23 @@ cpdef Table gather(
     Column gather_map,
     out_of_bounds_policy bounds_policy
 )
+
+cpdef Column shift(Column input, size_type offset, Scalar fill_values)
+
+cpdef Table scatter(object source, Column scatter_map, Table target_table)
+
+cpdef object empty_like(object input)
+
+cpdef Column allocate_like(Column input_column, mask_allocation_policy policy, size=*)
+
+cpdef Column copy_if_else(object lhs, object rhs, Column boolean_mask)
+
+cpdef Table boolean_mask_scatter(object input, Table target, Column boolean_mask)
+
+cpdef Column copy_range(
+    Column input_column,
+    Column target_column,
+    size_type input_begin,
+    size_type input_end,
+    size_type target_begin,
+)
```

Comment:
**Suggestion/nitpick** What order should write these? I would suggest one of:

1. Alphabetical
2. The same order as the C++ header file
3. The same order as the doxygen documentation of the matching header file (which I think is the same as (2)

Ideally we would also name the arguments the same, but that is very persnickity.

Reply:
I meant to do that, thanks for the reminder. I think matching the header makes sense.

Reply:
Done.